onellm/files.py

```
@@ -73,77 +161,159 @@ def upload(
+            # max_size is always set (defaults to 100MB if not specified)
+            file_size = None
+            is_seekable = False
+
+            # Try to get size from seekable file-like object
+            if hasattr(file, 'seek') and hasattr(file, 'tell'):
+                try:
+                    current_pos = file.tell()
+                    try:
+                        file.seek(0, 2)  # Seek to end
+                        file_size = file.tell()
+                        is_seekable = True
```

Flake8 (7.3.0)

[error] local variable 'is_seekable' is assigned to but never used

(F841)